src/lib/asn1/asn1_oid.cpp

```diff
@@ -88,20 +88,24 @@ OID OID::from_string(const std::string& str)
    if(!raw.empty())
       return OID(std::move(raw));
 
-   throw Lookup_Error("No OID associated with name " + str);
+   std::ostringstream err;
+   err << "No OID associated with name " << str;
+   throw Lookup_Error(err.str());
```

Comment:
Maybe not in this pull-request: But perhaps we want to start adopting `std::format` for things like that? (pending working compiler/library support)

Reply:
Not until GCC 13, which isn't even released yet :disappointed: